Popup window

luccamicale opened the pull request:
I create the window with a pop-up using javascript. The project is complete and works perfectly using javascript. I trust the work I do and that is why I would very much like to approve it in order to continue with the projects that I have left to finish the week.

vechicin commented on index.html:
```diff
@@ -75,7 +75,7 @@ <h1>Im Lucca
                 <button type="button">javascript</button>
```
This section should be created dynamically, please fix it.

<img width="1066" alt="Screen Shot 2022-09-15 at 15 29 05" src="https://user-images.githubusercontent.com/82426682/190502524-5e4b13dc-c9ad-4a73-9203-6f02587f4e44.png">

vechicin reviewed (changes requested):
# Hi @luccamicale ✋, 

Good job so far! 🔥 
There are some issues that you still need to work on to go to the next project but you are almost there! 🚨 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights ⚡ 

- [x] Good job implementing the modal window!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 📂 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

luccamicale commented:
I changed all things that told me the first reviewer. 
.I fix the image size
.I delete all elements of HTML and I put in Javascript
.I changed the bottons (similar to Figma)
.I accommodate the position of buttons

mateo951 reviewed (approved):
### Hi @luccamicale, 

Well done making all the changes requested by the previous reviewer ✔
Your project is complete!  There is nothing else to say other than... it's time to merge it 🐐 
### Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
